install.sh

```diff
@@ -213,7 +213,7 @@ function extract_install () {
       LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} tar -Ipixz -xpf ../"${2}"
     fi
     echo -e "${BLUE}Installing ${1} ...${RESET}"
-    tar cpf - ./*/* | (cd /; tar xp --keep-directory-symlink -f -)
+    cp -a ./usr/local/* "${CREW_PREFIX}"
```

Comment:
Why make this change? Tar is already on the system?

Reply:
[See above](https://github.com/skycocker/chromebrew/pull/6621#discussion_r784531492_)

Reply:
At the minimum, does `cp` preserve directory symlinks like the tar usage does?

Reply:
Yes, the `-a` option tell cp to preserve everything